### PR TITLE
fix: CU-201crra message sent not loading

### PR DESCRIPTION
**Pull request by LucasEstreladeOliveira**

### CODE REVIEW

File: packages/chat-sdk/src/channel/private-channel.ts [hunk 1]
```diff
@@ -443,6 +443,34 @@ export class PrivateChannel implements BasePrivateChannel {
...
+  /**
+   * Remove all private channel's listeners
+   *
+   */
+  public offAllListeners(): void {
+    this.messageModificationCallbacks[MessageChangeType.ADDED] = [];
+    this.messageModificationCallbacks[MessageChangeType.MODIFIED] = [];
+    this.messageModificationCallbacks[MessageChangeType.REMOVED] = [];
+
+    // this.reactionsAPI.offAllListeners();
```
Comment (cesarwbr) [inline packages/chat-sdk/src/channel/private-channel.ts:455]: remove this comment

**Review by cesarwbr — CHANGES_REQUESTED** [review]
check the comment below

File: packages/chat-sdk/src/channel/private-channel.ts (hunk 1 above)
Comment (LucasEstreladeOliveira) [reply packages/chat-sdk/src/channel/private-channel.ts:455]: ok!

**Review by cesarwbr — APPROVED** [review]
lgtm

**Pull request merged**